### ISSUE TITLE
Is there a tutorial for using it through docker?

### ISSUE THREAD

**Issue by sebastian0619** [Haste171/langchain-chatbot#17]
Hi there, I'm not so familiar with coding but i found this project extremely suit my needs. But for some reason i cannot build the project directly on the laptop i mainly work with (companies' asset). So is there a way that i can pack the project to a docker image and run it on my nas at home with access through web from my office?

I tried to deploy it via railway, everything seemed fine but after the deployment succeeded, it turned out with "Server Error" when i tried to access the through browser.
<img width="87" alt="image" src="https://github.com/Haste171/langchain-chatbot/assets/39752847/86d3d761-9568-4a6a-8644-a57265b0ea37">

**Comment by Haste171** [comment]
It has not been dockerized yet, i'm not exactly sure what your use case is but shouldn't be super hard to make it container compatible

**Comment by sebastian0619** [comment]
> And I wouldn't recommend running projects on a company laptop as they most likely restrict the amount of permissions interpreters like Python need

Yes, the purpose is to run the dockerised project on my NAS, so that I can access the project via internet remotely. And despite which device I'm using, I can query pdf through such web service (like on my company laptop).

**Comment by Haste171** [comment]
I can look into creating a docker contained version
